docs/content/_navigation.json

```diff
@@ -217,6 +217,14 @@
           {
             "title": "GraphQL Python Client",
             "path": "/concepts/dagit/graphql-client"
+          },
+          {
+            "title": "GraphQL Schema",
+            "pathTemplate": [
+              "https://github.com/dagster-io/dagster/blob/",
+              "/js_modules/dagit/packages/core/src/graphql/schema.graphql"
+            ],
```

Comment:
Happy to take other suggestions here - this is an external link that is version aware. My solution here is to interpolate version in between these two parts. Here, version serves as the tag to look for in github.